src/gerbil/compiler/driver.ss

```diff
@@ -28,7 +28,10 @@ namespace: gxc
   (getenv "GERBIL_GSC" "gsc"))
 
 (def gsc-runtime-args
-  ["-:f8,-8,t8"]) ;; force Gambit to use UTF-8
+  [;; force Gambit to use UTF-8:
+   (if (##unbound? (##global-var-ref (##make-global-var '##get-io-settings)))
+     "-:f8,-8,t8" ;; before v4.9.3-1081-g0680901f
+     "-:i8")]) ;; after
```

Comment:
You can do this with a `cond-expand` to avoid the runtime check:
```
(cond-expand
 (,(##unbound? ...)
   "-:f8,-8,-t8")
 (else "-:i8")))
```

Reply:
actually this might have issues with the bootstrap; nm, the cond-expand is fine.